templates/email/occurrence_cancelled-en.html

```diff
@@ -341,7 +341,7 @@
                                 <tr style="padding: 0; text-align: left; vertical-align: top;">
                                   <th style="Margin: 0; color: #0a0a0a; font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-weight: normal; line-height: 1.3; margin: 0; padding: 0; text-align: left;">
                                     <h2 style="Margin: 0; Margin-bottom: 10px; color: inherit; font-family: Helvetica, Arial, sans-serif; font-size: 30px; font-weight: normal; line-height: 1.3; margin: 0; margin-bottom: 10px; padding: 0; text-align: left; word-wrap: normal;">Cancellation notification</h2>
-                                    <h5 style="Margin: 0; Margin-bottom: 10px; color: inherit; font-family: Helvetica, Arial, sans-serif; font-size: 20px; font-weight: normal; line-height: 1.3; margin: 0; margin-bottom: 10px; padding: 0; text-align: left; word-wrap: normal;">Hello {{enrolment.person.name }}, unfortunately the event you were planning to participate in has been cancelled. </h5>
+                                    <h5 style="Margin: 0; Margin-bottom: 10px; color: inherit; font-family: Helvetica, Arial, sans-serif; font-size: 20px; font-weight: normal; line-height: 1.3; margin: 0; margin-bottom: 10px; padding: 0; text-align: left; word-wrap: normal;">Hello {{person.name }}, unfortunately the event you were planning to participate in has been cancelled. </h5>
```

Comment:
A trivial thing, but the H5 is ending with a whitespace.

Reply:
Fixed it 👍